accounting/fiscal_localizations/localizations/chile.rst

```diff
@@ -287,74 +287,50 @@ applicable to Purchase and Sales journals that can be related to the
 different sets of document types available in Chile. By default, all the sales journals created will
 use documents. 
   
-.. image:: media/Chile10.png
+.. image:: media/Chile41.png
    :align: center
    :alt: Document type configuration on Journal.
-
-Sequences
-*********
-
-When a Journal is created we have two options with document types sequences. 
-
-1. Create new sequences: In this case, Odoo will create new sequences for the most common document
-   types. 
-2. Use existent sequence: If the sequence should be shared between several journals, the user can
-   add manually sequences previously created.  
-
-.. image:: media/Chile11.png
-   :align: center
-   :alt: Steps to add new sequences on journals.
    
 .. important::
    For the Chilean localization it’s important to define the default Debt and Credit accounts 
    as they are required for one of the debit notes use cases.
    
 CAF
-***
+---
 
-For each sequence related to the documents type, a CAF (Folio Authorization Code) is required,
-the CAF is a file the SII provides to the Emisor with the folio/sequence authorized for the 
-electronic invoice documents.
+A CAF (Folio Authorization Code) is required, for each document type that is issued to your
+customers, the CAF is a file the SII provides to the Emisor with the folio/sequence authorized 
+for the electronic invoice documents.
 
 Your company can make several requests for folios and obtain several CAFs, each one associated
-with different ranges of folios. In this way, you could order folios per the different
-segmentations you have in your journals (example per office) thus avoiding distributing a range
-of folios. In this way, each branch would have its private key of the respective CAF.
+with different ranges of folios. In V14 the CAF are shared within all the journals, this means
+that you only need one active CAF per document type and it will be applied on all journals.
 
 Please refer to the SII documentation to check the detail on how to acquire the CAF:
 `SII Timbraje <https://palena.sii.cl/dte/mn_timbraje.html>`_.
 
 	
 Configuration
-^^^^^^^^^^^^^
-
-Once you have the CAF files you need to associate them with journal sequence in Odoo,
-after you create the sequence for the first time the will be in red, which indicates
-that no CAF has been configured for that particular sequence:
-
-.. image:: media/Chile12.png
```

Comment:
Is this file still used somewhere?

Reply:
No, it has been removed in the last commit